Include/internal/pycore_pythread.h

```diff
@@ -75,14 +76,22 @@ struct _pythread_runtime_state {
         struct py_stub_tls_entry tls_entries[PTHREAD_KEYS_MAX];
     } stubs;
 #endif
+
+    // Linked list of ThreadHandleObjects
+    struct llist_node handles;
```

Comment:
Unrelated, but shouldn't the `llist_*` symbols have a Py prefix?

Reply:
They are not public (`pycore_llist.h`) and do not export any symbols (all the inline functions are `static`).